### DISCUSSION TITLE
How to contribute?

### DISCUSSION THREAD

**Discussion by mknee40** [ProjectEvergreen/greenwood#594] [Q&A] [answered]
I like your static site generator, but wanted some direction on setting up local development environment. I forked your repository and tried running yarn develop or yarn serve nothing happens?

**Comment by mknee40** [comment, marked as answer]
Sorry, just found the .github/CONTRIBUTING.md file!

**Reply by thescientist13** [reply]
Hi and welcome!  👋 

I can also share [this page](https://www.greenwoodjs.io/getting-started/project-setup/) from our docs, which helps walk through some of those common project setup steps, and so maybe it could be of help to you as well.

Please let me know if there are any questions i can help with, either with contributing or starting a new project.  👍
